app/synchronizationmanager.cpp

```diff
@@ -230,6 +231,19 @@ void SynchronizationManager::onProjectSyncProgressChanged( const QString &projec
 
 }
 
+void SynchronizationManager::onProjectCreated( const QString &projectFullName, bool result )
+{
+  // 'projectFullName' is in the format "namespace/projectName" and
+  // 'mSyncProcess' stores projects in the format "projectName" only
```

Comment:
Just a minor adjustment, `mSyncProcesses` usually stores `namespace/projectName`, it is only when the project was previously not on the server altogether (went through the `createProject` method) that we use just the `projectName` form.

```suggestion
  // 'mSyncProcess' stores projects that were not previously uploaded to the server in the format "projectName".
```